context_test.go

```diff
@@ -595,6 +595,17 @@ func TestContextAbortWithStatus(t *testing.T) {
 	assert.True(t, c.IsAborted())
 }
 
+func TestContextAbortWithStatusJSON(t *testing.T) {
+	c, w, _ := CreateTestContext()
+	c.index = 4
+	c.AbortWithStatusJSON(415, "{\"foo\":\"bar\", \"barTwo\":\"fooTwo\"}")
+
+	assert.Equal(t, c.index, abortIndex)
+	assert.Equal(t, c.Writer.Status(), 415)
+	assert.Equal(t, w.Code, 415)
+	assert.True(t, c.IsAborted())
+}
```

Comment:
It is required to also test the output of that function, by checking `body == json{...}` and if `content_type == application/json`. Else looks good to me. Thanks @pjgg 

cc @appleboy